nucypher/cli/commands/deploy.py

```diff
@@ -62,6 +62,8 @@
 option_target_address = click.option('--target-address', help="Address of the target contract", type=EIP55_CHECKSUM_ADDRESS)
 option_gas = click.option('--gas', help="Operate with a specified gas per-transaction limit", type=click.IntRange(min=1))
 option_network = click.option('--network', help="Name of NuCypher network", type=click.Choice(NetworksInventory.networks))
+option_ignore_deployed = click.option('--ignore-deployed', help="Ignore already deployed contracts if exist.", is_flag=True)
+option_ignore_solidity = click.option('--ignore-solidity-check', help="Ignore solidity version compatibility check", is_flag=True, default=None)
```

Comment:
Minor thought: I would be a bit more explicit here for the name of the option - `option_ignore_solidity_version`.

Reply:
✔️